content/en/blog/_posts/2022-11-28-kubelet-credential-providers/index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: blog
+title: 'Kubernetes v1.26: GA Support for Kubelet Credential Providers'
+date: 2022-11-28
+slug: kubelet-credential-providers
+---
+
+**Authors:** Andrew Sy Kim (Google), Dixita Narang (Google)
+
+Kubernetes v1.26 introduced generally available (GA) support for _kubelet credential
```

Comment:
Do we want to link the docs here? i.e. /docs/tasks/kubelet-credential-provider/kubelet-credential-provider/
```suggestion
Kubernetes v1.26 introduced generally available (GA) support for _kubelet credential
```